keras_cv/layers/preprocessing/random_crop_and_resize_test.py

```diff
@@ -116,7 +116,7 @@ def test_aspect_ratio_factor_errors(self, aspect_ratio_factor):
     def test_crop_area_factor_errors(self, crop_area_factor):
         with self.assertRaisesRegex(
             ValueError,
-            "`crop_area_factor` must be tuple of two positive floats less than or "
+            "`crop_area_factor` must be tuple of two positive floats less than or"  # noqa
```

Comment:
noqa ?

Reply:
Actually earlier the length of line were exceeding the limit ,will remove them now